### DISCUSSION TITLE
PyCon India 2023 Keynote Speaker Suggestions

### DISCUSSION THREAD

**Discussion by bkamapantula** [pythonindia/inpycon2023-tasks#3] [General]
Please share your keynote speaker suggestions here. It should contain a brief description about the speaker or include a link to the bio of the speaker.

If you like a speaker suggestion, please vote by writing a line about your vote for e.g. "I would also love to hear X as a keynote speaker [because of these Y reasons]".

The selected keynote will be a mix of Indian and International speakers, so feel free to suggest from both.

We will start reaching out to keynote speakers in the order of preference within the next few weeks. Please suggest your keynote speakers by 15th April 23:59 IST.

**Comment by bpkapkar** [comment]
I wanted to suggest a few Python speakers who could add value to the event. These speakers are well-known in the Python community and have a wealth of knowledge to share

https://www.linkedin.com/in/denis-rothman-0b034043/ =>Has huge  contribution on NLP and Author of multiple AI books available.
https://www.linkedin.com/in/jessenoller => The 2021 Distinguished Service Award was presented to to Jesse Noller for his significant contributions to Python as a core developer, PyCon US Chair, and PSF board member.

**Comment by anuj0456** [comment]
Hi, I would like to propose a name for Guest Speaker.

Name: Swami Gopalakrishnan
Position: Director of NLP, Speech & LLM domain
Company: Accenture
Link to Bio: https://www.linkedin.com/in/swamigopalakrishnan1/

Swami has experience of 20+ years in technology.  He leads AI Architecture, AI for Speech, AI for Sustainability portfolios at Accenture India. He is also one of the leading architects in AI and custom engineering space and lead a diverse team involved in enabling AI at scale for clients. He has been instrumental in enabling AI for more than 30 clients in Accenture.

**Comment by adarshdigievo** [comment]
Name: Prashant Raghu
Linkedin: https://www.linkedin.com/in/prashanthraghu/

Prashant has written a free book on Python internals (https://github.com/PrashanthRaghu/advancedinternalsofcpython36book), listed in the official python-dev guide.

**Comment by adarshdigievo** [comment]
Name: Abhilash Raj
Linkedin: https://www.linkedin.com/in/rajabhilash/

CPython Core developer from India

**Comment by adarshdigievo** [comment]
Name: Karthikeyan Singaravelan
Linkedin: https://www.linkedin.com/in/karthikeyan-singaravelan-18373951

CPython Core developer from India

**Comment by adarshdigievo** [comment]
Name: Kushal Das
Linkedin: https://www.linkedin.com/in/kushal-das-5a655b8/

CPython Core developer from India

**Comment by adarshdigievo** [comment]
Name:  Marton Trencseni
Blog: https://bytepawn.com/tag/python.html

He has written some great intermediate/advanced articles related to Python (eg: https://bytepawn.com/building-a-simple-python-enum-class.html#building-a-simple-python-enum-class)

**Comment by adarshdigievo** [comment]
Name:  Tushar Sadhwani 
Blog: https://tushar.lol/

He has written many original and interesting Python topics in his blog (eg: https://tushar.lol/post/ast/).

**Reply by tusharsadhwani** [reply]
I'll have to agree 😆

**Reply by tusharsadhwani** [reply]
Here's an example for my talks:
https://youtu.be/oopv3b8oU_c

**Comment by abhiksark** [comment]
Sebastian Ramirez, the author of FastAPI, 
https://www.linkedin.com/in/tiangolo/
 Love to hear him as a keynote speaker because FastAPI is a powerful and fast framework for building APIs with Python. His insights on developing and scaling applications using FastAPI would be valuable for anyone interested in modern web development.

**Comment by abhiksark** [comment]
Guido van Rossum :) 
I think name is enough

**Comment by soumendrak** [comment]
Already many deserving speaker proposals are out there. If I may add one name, from whose contribution my debugging time and mostly other Python developers (> 3.10) got reduced is [Pablo Galindo Salgado](https://twitter.com/pyblogsal).

He is a:
- Python Steering Council and core developer. 
- Python 3.10/3.11 release manager and 
- also PSF Fellow. 

Due to his (and his team's) efforts, we got better tracebacks and faster 3.10 and 3.11. As a release manager, he can give us his first hand experience and more insights on the future of Python.

[More of his contributions](https://github.com/sponsors/pablogsal)

**Comment by kalyan678** [comment]
I would like to propose the following names for keynotes:

1. Peter Wang- CEO, Co-founder 
Anaconda,
 created 
pyscript_dev, 
PyData
 , Bokeh
, Datashader
. Former physicist. A student of the human condition. Opinions here are my own. Linkedin - https://www.linkedin.com/in/pzwang?original_referer= ,  Twitter - https://twitter.com/pwang?lang=en

2. Holden Karu- Holden is a transgender Canadian open source developer with a focus on Apache Spark, Airflow, Kubeflow, Ray, Dask and related big data tools.  Linkedin - https://www.linkedin.com/in/holdenkarau?original_referer=  , Twitter- https://twitter.com/holdenkarau

3. Travis Oliphant- Travis has worked extensively with Python for numerical and scientific programming since 1997, and was the primary developer of the NumPy package and the author of the definitive Guide to NumPy. He is also the primary founding author of the SciPy package. Linkedin - https://www.linkedin.com/in/teoliphant/ , Twitter- https://twitter.com/teoliphant?ref_src=twsrc%5Egoogle%7Ctwcamp%5Eserp%7Ctwgr%5Eauthor

All the above speakers are well known in the Python community. I wish to see at least one of them in the conference!!

**Comment by bkamapantula** [comment]
I'd like to nominate Harrison Chase.

Name: Harrison Chase
Profile: https://twitter.com/hwchase17
Known for: [LangChain](https://github.com/hwchase17/langchain)

In the last few years, LLMs have taken the world by storm. ChatGPT is one example LLM application. As open-source models get adopted and proprietary APIs get stabilized, production and community use-cases are slowly building up. The need for interfacing with LLMs has lead to several libraries.

LangChain, with first-class Python support, has emerged as a popular all-things-LLM-ops library. Harrison leads LangChain's community efforts. It'd be great to hear from him on what it means to build a library that has prolific releases (often multiple times a day), engaging the community and building emergent tools.

**Reply by abhi-glitchhg** [reply]
+1 for thiss

**Reply by aniketmaurya** [reply]
+1

**Comment by abhi-glitchhg** [comment]
I would like to nominate  Soumith Chintala 

profile: https://soumith.ch/
known for: PyTorch

Pytorch has become a de facto Deep Learning framework for research. Soumith was working on the PyTorch development from the beginning. I would really love to watch him as a keynote speaker.

**Reply by aniketmaurya** [reply]
+1 for this

**Comment by gpavanb1** [comment]
I would like to nominate Aakash N S

Name: Aakash N S
Known for: [Jovian](https://www.jovian.com/)

Aakash N S is the CEO and Lead Instructor at Jovian, an online education platform for Data Science and Machine Learning with over 350,000 registered learners. Courses taught by Aakash have over 6 million views on YouTube. Aakash has previously worked at Twitter as a software engineer and holds a bachelor’s degree in Computer Science from IIT Bombay

**Comment by ratulachakraborty** [comment]
I would like to nominate [Ananth Packkildurai](https://www.linkedin.com/in/ananthdurai/), who has gained widespread recognition for his blog [DataEngineeringWeekly](https://www.dataengineeringweekly.com/). Ananth has extensive knowledge of how Python is becoming increasingly prevalent in the creation of scalable Modern Data Stack architectures. Including Ananth's expertise in the conference will add a valuable dimension to the event by incorporating data, rather than making it overly focused on AI and backend topics.

**Comment by rohithcode** [comment]
I would like to nominate 

Prabhu Ramchandra - ( https://www.aero.iitb.ac.in/~prabhu/projects.html ) -
Janakiram MSV - Analyst and writes for Forbes - https://www.linkedin.com/in/janakiramm

**Comment by Vinaykumar-Puppala** [comment]
I would like to suggest as KEY note speaker **"Matei Zaharia"** [Matei Zaharia](https://github.com/mateiz)  / 
Linked IN:  https://www.linkedin.com/in/mateizaharia 
Email id : matei@cs.stanford.edu
Twitter : https://twitter.com/matei_zaharia

**Matei Zaharia, Co-founder and Chief Technologist; Original Creator of Apache Spark™ and MLflow at Databricks**

Matei Zaharia is a Cofounder and Chief Technologist at Databricks as well as an Assistant Professor of Computer Science at Stanford. He started the Apache Spark project during his PhD at UC Berkeley in 2009, and has worked broadly on other widely used data and machine learning software, including MLflow, Delta Lake and Apache Mesos.

**Reply by Vinaykumar-Puppala** [reply]
@bkamapantula  - fyi!

**Comment by Vinaykumar-Puppala** [comment]
I would like to suggest one other keynote speaker **Chip Huyen**  (Git Hub: https://github.com/chiphuyen) 

Chip Huyen, a writer and computer scientist. She building infrastructure or real-time ML. also teach Machine Learning Systems Design at Stanford and Author of "**Designing Machine Learning System**s" Best Seller and talk about real-time LLM  for production (https://huyenchip.com/2023/04/11/llm-engineering.html)

Email : [chip@huyenchip.com](mailto:chip@huyenchip.com)
Twitter : https://twitter.com/chipro
Linked in : https://www.linkedin.com/in/chiphuyen

**Reply by Vinaykumar-Puppala** [reply]
@bkamapantula - fyi!

**Reply by aniketmaurya** [reply]
+1 for this

**Comment by Vinaykumar-Puppala** [comment]
Sandeep Alur , Director - Microsoft Technology Center
(AI Enthusiast, an Aspiring Data Scientist & a Cloud Practitioner. Driving Technology Strategy for Customers in India.)

TECHNICAL LEADERSHIP: Led large & complex projects for customers in North America, UK, and India. A result-oriented person who has built High Performing Teams
TECHNOLOGY SKILLS: Azure & AWS Cloud, Big Data Stack, Machine Learning, Deep Learning, Data Mining, Algorithms.

Linked In : https://www.linkedin.com/in/sandeepalur/

**Comment by Vinaykumar-Puppala** [comment]
Debdoot Mukherjee, Head of AI, Demand Engg @ Meesho
Linked In: https://www.linkedin.com/in/debdootmukherjee/

Debdoot has 14+ years of experience in AI/ML/Data Science, Currently, he is working at Meesho as Head Of AI. 
Previously, I led the **AI team at ShareChat,** and **Data Sciences @ Hike** and  graduated from Indian Institute of Technology (IIT)

**Specialties:** Recommender Systems, Information Retrieval, User Modeling, Personalization, Information Extraction, Machine Learning, Social Graph Mining, Conversational Modeling, Enterprise Search, and Context-based applications.

**Comment by Vinaykumar-Puppala** [comment]
_For workshop Speaker Suggestion:_  **Nitin Aggarwal,  Head of Cloud AI Services (India) at Google**
 LinkedIn : https://www.linkedin.com/in/ntnaggarwal/
 Info : Machine learning and Artificial intelligence (AI) professional with 10+ years of Industry experience in Deep learning, Data Science, Business Analytics, Management Consulting, Product Management and Software development. I'm currently leading Cloud AI and Industry Solution Services team in India. This team is responsible to design AI solutions/products for Google's strategic enterprise customers. (FYI :https://www.linkedin.com/in/ntnaggarwal/overlay/1635469176841/single-media-viewer/?profileId=ACoAAAOpidsBvZgJni_7v_0L6h13DN4EgsT3u8w)  
 reference : https://analyticsindiamag.com/40-under-40-data-scientists-in-india-2021-who-are-they/
 
_Technical & Functional capabilities:_
Business Analytics, Statistical Analysis and Data Mining, Consulting, Growth hacking, Data Visualization, Machine Learning/Artificial Intelligence (Structured & unstructured - Text, Image & Video), Customer & People Analytics, Agile project management, Product Management

_Technology strongholds:_
Python, R, TensorFlow, Keras, Advance SQL, Base SAS, Alteryx, Tableau, Google Cloud, AWS, Kubernetes, KubeFlow, PyTorch

**Comment by aniketmaurya** [comment]
I would also love to hear [Anima Anandkumar](https://www.linkedin.com/in/anima-anandkumar/) as a keynote speaker. She is leading AI research at Nvidia.

**Comment by aniketmaurya** [comment]
I would love to hear from [Yannic Kilcher](https://www.youtube.com/@YannicKilcher). He is leading OpenAssistant, an open-source ChatGPT-like model. He also open-sourced the dataset.

**Comment by tusharsadhwani** [comment]
I'd like to see [Yury Selivanov](https://twitter.com/1st1), the creator of asyncio library in Python, one of the people behind the async await syntax, and creator of uvloop, asyncpg and edgedb.

**Comment by bkamapantula** [comment]
Thank you all for the great suggestions! Closing the discussion as the deadline has passed. The team will discuss these options next.
